deploy/deploy.js

```diff
@@ -1,5 +1,8 @@
 const fs = require('fs');
-require('dotenv').config();
+const path = require('path')
+require('dotenv').config({
+  path: path.join(__dirname, '.env')
+});
 
 async function deployNativeToErc(){
   const deployHome = require('./src/native_to_erc/home');
```

Comment:
If there is no usage of `__dirname` in `require('./src/native_to_erc/home');` does it mean that `require` works with paths in different manner?

Reply:
Yes, `require` is always relative to the file, so you don't need to do the `path.join(__dirname, ...)` trick.